docs/community/contribute/contribution-tutorial.md

```diff
@@ -93,9 +93,9 @@ GitHub will check if your changes create any merge conflicts with the branch you
 
 ## Waiting for a review
 
-Before your changes can be merged into the project, they have to pass a review. Some other IPFS repose, like `ipfs/ipfs-js` and `ipfs/ipfs-go`, have automatic tests that run against a pull request. These tests must pass _before_ the changes can be merged into the project.
+Before your changes can be merged into the project, they have to pass a review. Some other IPFS repos, like `ipfs/ipfs-js` and `ipfs/ipfs-go`, have automatic tests that run against a pull request. These tests must pass _before_ the changes can be merged into the project.
```

Comment:
```suggestion
Before your changes can be merged into the project, they have to pass a review. Some other IPFS repositories, like `ipfs/ipfs-js` and `ipfs/ipfs-go`, have automatic tests that run against a pull request. These tests must pass _before_ the changes can be merged into the project.
```

Nice catch! We should probably use _repositories_ rather than _repos_.